src/graphnet/training/loss_functions.py

```diff
@@ -33,10 +33,11 @@ def __init__(self, **kwargs: Any) -> None:
     @final
     def forward(  # type: ignore[override]
         self,
-        prediction: Tensor,
+        predictions: Tensor,
         target: Tensor,
```

Comment:
here set the `target: Tensor = None` then it is not needed to specify the value `None` in the Task